app/src/pages/guestbook/[id]/edit.tsx

```diff
@@ -23,27 +23,28 @@ export default function GuestBookEdit() {
     } = useGuestBookMessage(router.query.id as string);
     const detailsHref = `/guestbook/${message?.messageId}`;
 
-    const [draftMessage, setDraftMessage] = useState<IGuestBookMessage>();
-    const isLoading = isMessageLoading || isUndefined(draftMessage);
+    // const [draftMessage, setDraftMessage] = useState<IGuestBookMessage>();
+    // const isLoading = isMessageLoading || isUndefined(draftMessage);
```

Comment:
Nice, I'm glad you were able to refactor the existing state management so that this is no longer necessary!